docs/config.md

```diff
@@ -400,11 +400,14 @@ deleted if you push the branch with `jj git push --branch` or `jj git push
 
 # Alternative ways to specify configuration settings
 
-Instead of `~/.jjconfig.toml`, the config settings can be located at
-`$XDG_CONFIG_HOME/jj/config.toml` as per the [XDG specification]. It is an error
-for both of these files to exist.
-
-[XDG specification]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+Instead of `~/.jjconfig.toml`, the config settings can be located under
+a platform-specific directory. It is an error for both of these files to exist.
+
+| Platform | Value                                  | Example                                                   |
+| :------- | :------------------------------------- | :-------------------------------------------------------- |
+| Linux    | `$XDG_CONFIG_HOME/jj`                  | `/home/alice/.config/jj/config.toml`                      |
```

Comment:
<small>noticed the rewrite</small>

As I've said on the other comment, by your logic the `config.toml` suffix should be removed from other platforms as well.

However I think that docs should reflect the current state of the program, not a potential future, and when/if that todo gets implemented this doc would be updated (e.g. to have `*.toml` in place of `config.toml`).